modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+import glob
+
+# Method used for evaluate corners coords 
+# Not used yet - need to re-write calculations with it
+def calc_coords(bb):
+    xmin = bb[0]
+    xmax = bb[0] + bb[2] - 1.0
+    ymin = bb[1]
+    ymax = bb[1] + bb[3] - 1.0
+    cx = bb[0] + (bb[2] + 1.0) / 2
+    cy = bb[1] + (bb[3] + 1.0) / 2
+    coords = {'xmin': xmin, 'xmax': xmax, 'ymin': ymin, 'ymax': ymax,
+                   'cx': cx, 'cy': cy}
+    return coords
+
+# Method used for initializing of trackers by creating it 
+# via cv.TrackerX_create()
+def init_tracker(tracker_name):
+    config = {"Boosting": (cv.TrackerBoosting_create(), 500),
+    "MIL": (cv.TrackerMIL_create(), 1000),
+    "KCF": (cv.TrackerKCF_create(), 250),
+    "MedianFlow": (cv.TrackerMedianFlow_create(), 500),
+    "GOTURN": (cv.TrackerGOTURN_create(), 250),
+    "MOSSE": (cv.TrackerMOSSE_create(), 250),
+    "CSRT": (cv.TrackerCSRT_create(), 250)}
+    return config[tracker_name]
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Run LaSOT-based benchmark for visual object trackers")
+    # As a default argument used name of 
+    # original dataset folder
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="LaSOTTesting", help="Full path to LaSOT")
+    parser.add_argument("--visualization", type=str,
+                        default=False, help="Showing process of tracking on video")
```

Comment:
`parser.add_argument("--visualization", action='store_true', help="Showing process of tracking on video")`

Reply:
OK.